core/templates/domain/topic/topic-summary.model.ts

```diff
@@ -154,6 +175,18 @@ export class TopicSummary {
     return this.thumbnailBgColor;
   }
 
+  getDegreesOfMastery(): DegreesOfMastery {
+    return this.degreesOfMastery;
+  }
+
+  getSkillDescription(): SkillIdToDescriptionMap {
```

Comment:
Is the return type correct here?

Reply:
Yeah, this function returns `[skillId: string]: string` which is SkillIdToDescriptionMap type.